server/config.js

```diff
@@ -38,6 +38,8 @@ let dashboardConfig = {
 
 let serverConfig = {
     'PORT': '6454',
+    'PULSAR_URL': 'http://localhost:8080',
+    'WEBSOCKET_URL': 'https://websocket.example.com:8500',
     'LOG_LEVEL': 'info',
```

Comment:
To completely remove nginx, the server will also have to support TLS. Previously, nginx was providing that ability. There are default certs used by nginx and the vue dev server that can be used to test TLS configuration.

Reply:
Actually, it looks like the server might already support TLS.